File: CursoPOOUber/Java/Car.java
public class Car {
    Integer id;
    String license;
    Account driver;
    Integer passegenger;

    public Car(String license, Account driver) {
        this.license = license;
        this.driver = driver;
    }

    void printDataCar() {
        System.out.println("Id: " + id);
        System.out.println("Car " + id + " License: " + license);
        System.out.println("Car " + id + " Driver: " + driver.name);
        System.out.println("Car " + id + " Passegenger: " + passegenger);
    }
}

File: CursoPOOUber/JS/car/car.js
class Car {
    #passegenger;
    constructor(license, driver) {
        this.id;
        this.license = license;
        this.driver = driver;
        this.#passegenger = 0;
    }

    get passegengers () {
        return `The passegenger/s is/are: ${this.#passegenger} `;
    }

    set passegengers (amount) {
        amount === 4 
        ? this.#passegenger = amount
        : console.log('Tienen que ser cuatro pasajeros.');
    }
    
    printDataCar() {
        let {driver} = this;
        console.group('Driver');
        console.table(driver);
        console.groupEnd();
    }
};